client/templates/topic/topicInfoItemEdit.js

```diff
@@ -237,7 +237,8 @@ Template.topicInfoItemEdit.events({
         cancelButton.prop('disabled',false);
 
         let editItem = getEditInfoItem();
-        tmpl.find('#id_item_subject').value = (editItem) ? editItem._infoItemDoc.subject : '';
+        let infoItem = tmpl.find("#id_item_subject");
```

Comment:
infoItem is not a good name for this variable.
A few lines down you named the same thing with a better name: `let itemSubject = tmpl.find('#id_item_subject');` ;-)